std/format/write.d

```diff
@@ -51,22 +51,28 @@ Source: $(PHOBOSSRC std/format/write.d)
 module std.format.write;
 
 /**
- * `bool`s are formatted as `"true"` or `"false"` with `%s` and as `1` or
- * `0` with integral-specific format specs.
+`bool`s are formatted as `"true"` or `"false"` with `%s` and like the
+`byte`s 1 and 0 with all other format characters.
```

Comment:
I think it sounds better if we drop the `byte` mentione. E.g: "[...] and as 1 or 0 with all other format characters."

Reply:
I added that, because of `%r`. That's the only difference (see #7946).